tests/components/samsungtv/test_config_flow.py

```diff
@@ -1296,7 +1279,6 @@ async def test_update_old_entry(hass: HomeAssistant) -> None:
     entry2 = config_entries_domain[0]
 
     # check updated device info
-    assert entry2.data.get(CONF_IP_ADDRESS) is not None
```

Comment:
Seems we are no more testing for a change of network config

Reply:
> Seems we are no more testing for a change of network config

That is not linked to this PR